generators/php/sdk/src/endpoint/http/HttpEndpointGenerator.ts

```diff
@@ -94,6 +108,16 @@ export class HttpEndpointGenerator extends AbstractEndpointGenerator {
         });
     }
 
+    private getClassReference(requestBody: HttpRequestBody): php.ClassReference {
```

Comment:
When we update the naming above, let's name this `getRequestTypeClassReference`.

Reply:
done